src/pages/home/Home.jsx

```diff
@@ -1,70 +1,85 @@
 import "./home.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
 
-export const Home = () => (
-  <>
-    <main className="flex-col">
-      {/* <!-- block-1 --> */}
-      <div className="display-block">
-        <div className="flex-row landscape-overcard">
-          <img
-            src="/assets/images/sofaBeige.webp"
-            alt="living room decor"
-            className="responsive-img"
-          />
-          {/* <!-- over-card --> */}
-          <div className="overlay-solid-card med-text">
-            Welcome to your one stop shop for elegant and chic Decor.
-          </div>
-        </div>
-      </div>
-      {/* <!-- end of block-1 --> */}
+export const Home = () => {
+  useEffect(() => {
+    (async function () {
+      try {
+        let category = await axios.get("/api/categories");
+        setCategories(category.data.categories);
```

Comment:
you can destructure the object from get method example 
await axios.get("/api/categories")
.then(({data})=>{
  const {categories} =data;
setCategories(categories);
})

Reply:
why would await and .then be used together, don't they serve the same purpose?